Makefile

```diff
@@ -10,7 +10,7 @@ clean:
 	rm -rf dist venv .tox *.egg-info *.log*
 
 venv:
-	/usr/bin/env python3 -m venv venv
+	/usr/bin/env python -m venv venv
```

Comment:
This change is unnecessary + we specifically DO want to enforce **some** python3 installation (i.e. respect the python3 symlink that is compatible with `update-alternatives`) and we want to be absolutely open and clear about it, so please keep the original.

Reply:
That wasn't intended; I just hit the backspace one more time.